leanplum_data_export/get_messages.py

```diff
@@ -0,0 +1,80 @@
+"""Get all messages from leanplum API and save to bigquery."""
+
+import datetime
+import json
+
+import requests
+from google.cloud import bigquery
+
+LEANPLUM_API_URL = "https://api.leanplum.com/api"
+# latest version can be found at https://docs.leanplum.com/reference#get_api-action-getmessages
+LEANPLUM_API_VERSION = "1.0.6"
+
+
+class LeanplumMessageFetcher(object):
+    def __init__(self, app_id, client_key, project, bq_dataset, table_prefix, version):
+        self.app_id = app_id
+        self.client_key = client_key
+        self.project = project
+        self.bq_dataset = bq_dataset
+        self.table_prefix = table_prefix
+        self.version = version
+
+    def write_to_bq(self, date, messages):
+        bq_client = bigquery.Client(project=self.project)
+
+        table_name = f"messages_v{self.version}"
+        if self.table_prefix is not None:
+            table_name = f"{self.table_prefix}_{table_name}"
+
+        load_config = bigquery.LoadJobConfig(
+            time_partitioning=bigquery.TimePartitioning(
+                field="load_date",
+                require_partition_filter=True,
```

Comment:
Not sure we need a partition filter for such a small table.

Reply:
I added this because for analysis you would probably only look at one day and this makes it so that you can't accidentally load duplicate messages from every day.  I'm open to removing this though.